airflow/dags/payments_views/sbmtd_transactions.sql

```diff
@@ -0,0 +1,86 @@
+---
+operator: operators.SqlToWarehouseOperator
+dst_table_name: "views.sbmtd_transactions"
+external_dependencies:
+  - payments_loader: all
+---
+
+with
+
+/*
+  All the unique routes and stops across SBMTD
+*/
+route_stops as (
+    select distinct
+        route_id,
+        stop_id
+    from `cal-itp-data-infra.gtfs_schedule.stop_times` as st
+    join `cal-itp-data-infra.gtfs_schedule.trips` as t using (calitp_itp_id, trip_id)
```

Comment:
calitp_itp_id AND calitp_url_number together identify a feed, so both need to be included in the USING clause. (sorry for this wart in the data model :/ )